src/2d/spreadinterp2d.cu

```diff
@@ -57,7 +57,7 @@ void eval_kernel_vec_Horner(FLT *ker, const FLT x, const int w,
 	FLT z = 2*x + w - 1.0;         // scale so local grid offset z in [-1,1]
 	// insert the auto-generated code which expects z, w args, writes to ker...
 	if (upsampfac==2.0) {     // floating point equality is fine here
-#include "../../finufft/ker_horner_allw_loop.c"
+#include "../../contrib/ker_horner_allw_loop.c"
```

Comment:
Can this move up top with the other includes?

Reply:
Since this file is from finufft, I am not sure if it is a good idea to put it at other place.

Reply:
I see. I didn't realize this was some sort of macro trick.